internal/command/user_v2_phone_test.go

```diff
@@ -873,8 +903,7 @@ func TestCommands_changeUserPhoneWithGenerator(t *testing.T) {
 		{
 			name: "phone changed, return code",
```

Comment:
Also a test with the external code to change the phone?

Reply:
done